cpp/unreal_plugins/SpCore/Source/SpCore/UnrealClassRegistrar.h

```diff
@@ -42,6 +42,18 @@ class SPCORE_API UnrealClassRegistrar
     static void initialize();
     static void terminate();
 
+    //
+    // Get static class using a class name instead of template parameters
+    //
+
+    static UClass* getStaticClass(const std::string& class_name);
+
+    //
+    // Get Default object using a class name instead of template parameters
+    //
+
+    static UObject* getDefaultObject(const std::string& class_name);
```

Comment:
Let's order these functions as follows:

```
newObject
loadObject
getDefaultObject
getStaticClass
```

I think this is a slightly more logical ordering, because then the return types of of all our create/get functions will be sorted roughly in high-level-to-low-level order (`AActor`, `UComponent`, `UObject`, `UClass`).